### PR TITLE
updated Admin docs to include guidance on third party interoperability

### PR DESCRIPTION
**Pull request by matthewhegarty**
**Problem**

We've had a few issues related to interoperability with 3rd party libraries.  Rather than trying to patch this, I suggest that documenting workarounds is the best solution for now.

**Solution**

Updated Admin UI docs and FAQ.

**Acceptance Criteria**

- unit tests
- reviewed docs locally

### CODE REVIEW

File: docs/advanced_usage.rst [hunk 1]
```diff
@@ -841,6 +841,31 @@ return books for the publisher::
...
+Interoperability with 3rd party libraries
+-----------------------------------------
+
+import_export extends the Django Admin interface.  There is a possibility that clashes may occur with other 3rd party
+libraries which also use the admin interface.
+
+django-admin-sortable2
+^^^^^^^^^^^^^^^^^^^^^^
+
+Issues have been raised due to conflicts with setting `change_list_template <https://docs.djangoproject.com/en/dev/ref/contrib/admin/#django.contrib.admin.ModelAdmin.change_list_template>`_.  There is a workaround listed `here <https://github.com/jrief/django-admin-sortable2/issues/345#issuecomment-1680271337>`_.
```
Comment (EricOuma) [inline docs/advanced_usage.rst:855]: I think it's better to use the stable version link:
https://docs.djangoproject.com/en/stable/ref/contrib/admin/#django.contrib.admin.ModelAdmin.change_list_template

Comment (matthewhegarty) [reply docs/advanced_usage.rst:855]: Done

Comment (EricOuma) [reply docs/advanced_usage.rst:855]: The update commit is yet to come

Comment (matthewhegarty) [reply docs/advanced_usage.rst]: Apologies - is there now

**Review by EricOuma — APPROVED** [review]

**Pull request merged**